package.json

```diff
@@ -11,16 +11,30 @@
         "zerorpc": "latest"
     },
     "devDependencies": {
+        "@types/chai": "^4.1.5",
+        "@types/chai-as-promised": "^7.1.0",
+        "@types/datatables.net": "^1.10.13",
+        "@types/jquery": "^3.3.6",
+        "@types/jquery.contextmenu": "^1.7.33",
+        "@types/mocha": "^5.2.5",
+        "@types/node": "^8.9.2",
+        "@types/tail": "^1.2.0",
         "chai": "^4.1.2",
         "chai-as-promised": "^7.1.1",
-        "electron": "1.8.4",
+        "electron": "^1.8.4",
```

Comment:
Can you leave it to `1.8.4` for now? As discussed I wanna specifically update electron alone at some point (soon). But until then it should be pinned to a single version.